<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    border: 1px solid black; /* Цвет и толщина рамки */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 5px; /* Отступы внутри layout */
    color: black; /* Цвет текста */
	font-family: 'Bahnschrift', sans-serif;
}

/*BTN*/
QPushButton {
    background-color: #2A9DF4; /* Цвет фона */
    color: white; /* Цвет текста */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 5px; /* Отступы внутри кнопки */
}
QPushButton:hover {
    background-color: #1877F2; /* Цвет фона при наведении */
}
QPushButton:pressed {
    background-color: #103E91; /* Цвет фона при нажатии */
}
/*-BTN*/


QSlider::groove:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    border: 1px solid #999999;
    height: 8px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* отступы от верхнего и нижнего краев слайдера */
    border-radius: 4px;
}

QSlider::groove:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #B1B1B1, stop:1 #c4c4c4);
    border: 1px solid #999999;
    width: 8px; /* Ширина грани */
    border-radius: 4px; /* Скругление углов */
}

QSlider::handle:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eeeeee, stop:1 #cccccc);
    border: 1px solid #5c5c5c;
    height: 18px; /* Высота ползунка */
    margin: 0 -2px; /* Отступы от левого и правого краев слайдера */
    border-radius: 4px; /* Скругление углов */
}

QWidget {
	background-color: rgb(0, 10, 90); /* Цвет фона */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="QWid" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
    border: 1px solid black; /* Цвет и толщина рамки */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 10px; /* Отступы внутри layout */
	background-color: rgb(0, 100, 100); /* Цвет фона */
    color: black; /* Цвет текста */
}

QPushButton {
    background-color: #2A9DF4; /* Цвет фона */
    color: white; /* Цвет текста */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 5px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: #1877F2; /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #103E91; /* Цвет фона при нажатии */
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>ЛАБОРОТОРНЫЕ РАБОТЫ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>319</width>
           <height>114</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="8" column="2">
           <widget class="QPushButton" name="pushButton_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-6</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QPushButton" name="pushButton_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-8</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4">
           <widget class="QPushButton" name="pushButton_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-4</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-3</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QPushButton" name="pushButton_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-7</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-2</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-1</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-5</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pushButton_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-9</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QPushButton" name="pushButton_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-10</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3">
           <widget class="QPushButton" name="pushButton_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-11</string>
            </property>
           </widget>
          </item>
          <item row="9" column="4">
           <widget class="QPushButton" name="pushButton_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ЛР-12</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton_13">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    border: 1px solid black; /* Цвет и толщина рамки */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 5px; /* Отступы внутри layout */
    color: black; /* Цвет текста */
	font-family: 'Bahnschrift', sans-serif;
}

/*BTN*/
QPushButton {
    background-color: #2A9DF4; /* Цвет фона */
    color: white; /* Цвет текста */
    border-radius: 10px; /* Радиус скругления углов */
    padding: 5px; /* Отступы внутри кнопки */
}
QPushButton:hover {
    background-color: #1877F2; /* Цвет фона при наведении */
}
QPushButton:pressed {
    background-color: #103E91; /* Цвет фона при нажатии */
}
/*-BTN*/</string>
     </property>
     <property name="text">
      <string>Вернуться</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
